WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.33.0"
+  s.version       = "1.33.1-beta.1"
```

Comment:
We leave the patch version for hotfixes, so this should be `1.34.0-beta.1`.

Reply:
Great catch! Updated the pod version in: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/562/commits/30084d93f0ae76012b1337d7d25b2421cf759a06